test/fast-list-test.js

```diff
@@ -109,6 +109,13 @@ suite('FastList >', function() {
       sinon.assert.calledWith(scheduler.attachDirect, container, 'scroll');
     });
 
+    test('it calls source.getViewportHeight() if provided', function() {
+      source.getViewportHeight = sinon.stub();
+      source.getViewportHeight.returns(400);
+      fastList = new FastList(source);
+      sinon.assert.called(source.getViewportHeight);
```

Comment:
I'd love to assert on the `400` somewhere but can't find an elegant way to do it without testing internals. Not a big deal anyway.